src/structure/structure.ts

```diff
@@ -439,9 +446,9 @@ export class StructureViewer {
             close.classList.add('chsp-close-widget-button', 'btn', 'btn-light', 'btn-sm');
             close.id = `chsp-close-widget-button-${cellGUID}`;
             close.onclick = () => {this._removeWidget(cellGUID); this._setupGrid(this._selected.size); };
-            close.innerHTML = `
+            close.innerHTML = `<i class="fa fa-times"></i>
```

Comment:
I would rather not do that, font awesome is available on the default viewer but not a dependency of the core library. We could use svg of the corresponding fa glyphs though.

Reply:
OK! Will try and fix these as soon as I have the time, but if you want to push a fix in the meantime, feel free